sdks/java/io/bigquery-io-tests/src/test/java/org/apache/beam/sdk/bigqueryiotests/BigQueryIOIT.java

```diff
@@ -94,15 +96,20 @@
 
     void setSourceOptions(String value);
 
-    @Description("BQ dataset with source data")
-    String getBigQuerySourceDataset();
+    @Description("BQ dataset for the test data")
+    String getBigQueryTestDataset();
 
-    void setBigQuerySourceDataset(String dataset);
+    void setBigQueryTestDataset(String dataset);
 
-    @Description("BQ table with source data")
-    String getBigQuerySourceTable();
+    @Description("BQ table for test data")
+    String getBigQueryTestTable();
 
-    void setBigQuerySourceTable(String table);
+    void setBigQueryTestTable(String table);
+
+    @Description("Should test use streaming writes or batch loads to BQ")
+    Boolean getUseStreamingWrites();
+
+    void setUseStreamingWrites(Boolean value);
```

Comment:
Can we use String or the enum itself for that option? So that users could explicitly tell: 
`--writeMethod=STREAMING_WRITES` or `--writeMethod=BATCH_LOADS`

This is similar to TextIOIT case and compression type case: [TextIOIT](https://github.com/apache/beam/blob/e0cbd2aa7e371c75511544ab78075d54f3f086ca/sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/text/TextIOIT.java#L98)

Possible alternative: I never tried this but maybe it makes sense to use [InstanceFactory](https://github.com/apache/beam/blob/279a05604b83a54e8e5a79e13d8761f94841f326/sdks/java/core/src/main/java/org/apache/beam/sdk/options/Default.java#L145) here?

Reply:
I didn't think of it, I changed it to use a String for the option, taking an enum value